src/AnyTVKue.js

```diff
@@ -4,16 +4,32 @@ import _ from 'lodash';
 import winston from 'winston';
 import baseConfig from './config';
 import kue from 'kue-scheduler';
-import basic_auth from 'basic-auth-connect'
-
+import basic_auth from 'basic-auth-connect';
+
+/**
+ * AnyTVKue class injects additional helper functions on kue-scheduler
+ * @example
+ * const kue = require('anytv-kue')(config.app.KUE);
+ * const Queue = kue.createQueue(config.app.KUE_OPTIONS);
+ * kue.setup(Queue); // adds error listeners and requeue previous jobs
```

Comment:
Might be better to just reference the doc then? Since we may be adding more than just adding error listeners/requeue previous jobs?